zeebe/topology/src/main/resources/proto/requests.proto

```diff
@@ -31,6 +31,7 @@ message LeavePartitionRequest {
 message ScaleRequest {
   repeated string memberIds = 1;
   bool dryRun = 2;
+  optional int32 newReplicationFactor = 3; // can be -1 to keep the current replication factor
```

Comment:
:thought_balloon: I think it would be cleaner to rely on the `optional`, i.e. not set this field to keep the current replication factor.. Then this field could also be `uint32`.